pvlib/irradiance.py

```diff
@@ -31,6 +30,90 @@
                    'sea': 0.06}
 
 
+def cloud_opacity_factor(irr_dif_clouds: np.ndarray,
+                         irr_dir_clouds: np.ndarray,
+                         irr_ghi_clouds: np.ndarray,
+                         spectra: dict) -> (np.ndarray, np.ndarray):
+    """
+        Calculate the effect of "cloud opacity factor" on spectral
+        irradiance under clear sky.
+
+        First we calculate the rho fraction based on campbell_norman
+        irradiance with clouds converted to POA irradiance. In the
+        paper [1] these values are obtained from observations. The equations
```

Comment:
```suggestion
        paper [1]_ these values are obtained from observations. The equations
```

Adding an underscore like this turns the [1] into a clickable link to the reference text below